packages/backend/scripts/createOpStack.ts

```diff
@@ -0,0 +1,214 @@
+// eslint-disable-next-line import/no-internal-modules
+import { RawDiscoveryConfig } from '@l2beat/discovery/dist/discovery/config/RawDiscoveryConfig'
+import { assert, EthereumAddress } from '@l2beat/shared-pure'
+import { exec } from 'child_process'
+import { mkdir, readFile, writeFile } from 'fs/promises'
+import path from 'path'
+import util from 'util'
+
+const execPromise = util.promisify(exec)
+
+interface HelpMode {
+  mode: 'help'
+}
+
+interface OpStackMode {
+  mode: 'opStack'
+  project: string
+  initialAddresses: [EthereumAddress, EthereumAddress]
+}
+
+type Config = HelpMode | OpStackMode
+
+void main().catch((e) => {
+  console.error(e)
+  process.exit(1)
+})
+
+async function main() {
+  const config = parseCliParameters()
+  switch (config.mode) {
+    case 'help':
+      printUsage()
+      break
+    case 'opStack':
+      await createOpStackProject(config.project, config.initialAddresses)
+      break
+  }
+}
+
+async function createOpStackProject(
+  project: string,
+  initialAddresses: EthereumAddress[],
+): Promise<void> {
+  const dir = path.join(__dirname, '..', 'discovery', project, 'ethereum')
```

Comment:
There should be a small check that the resulting path is the one you'd expect. If I move this script or the discovery directory everything is going to suck big time